gluster_exporter/main.go

```diff
@@ -33,7 +35,24 @@ func registerMetric(name string, fn func(), intervalSeconds int64) {
 }
 
 func getPeerID() string {
-	return *peerid
+	gdInfo, err := os.Open("/var/lib/glusterd/glusterd.info")
```

Comment:
Added comment to make it compatible with glusterd1 and glusterd2. https://github.com/gluster/gluster-prometheus/issues/31#issuecomment-422648729

Also please cache the output so that only looks for actual files if not found.

    func getPeerID() string {
        if peerid != "" {
             return peerid
        }
        peerid = parse_toml_or_info_file()
        return peerid
    }